sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.4.0-beta.1 (Unreleased)
+## 12.4.0 (2022-03-09)
```

Comment:
Is this when we are releasing or on 03-08?

Reply:
That is just estimated date for release. We can adjust the date if necessary.